packages/tizen_app_control/example/lib/main.dart

```diff
@@ -185,12 +185,12 @@ class _MyAppState extends State<MyApp> {
             children: <Widget>[
               ElevatedButton(
                 onPressed: _sendSms,
-                child: const Text('Send SMS'),
+                child: const Text('Send SMS(No TV)'),
```

Comment:
Can we find a better example for TV? (as we are deprecating the wearable profile)

Reply:
I removed that did not work. And I found 'web page open' is works. so i added it.